docs/integrations/orms/mikroorm.md

```
@@ -90,6 +90,41 @@ export class AppModule {}
+  @Property({ formula: (cols) => `LOWER(${cols.title})` })
+  titleLower!: string;
+}
+```
+
+```ts
+@Kavo(Book, {
+  allowlists: { filterable: ["titleLower"], sortable: ["titleLower"], selectable: ["id", "title", "titleLower"] },
+})
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)